src/node_v8.cc

```diff
@@ -60,6 +60,13 @@ void GetHeapStatistics(const FunctionCallbackInfo<Value>& args) {
 
 
 void SetFlagsFromString(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() < 1)
+    return env->ThrowTypeError("v8 flag is required");
```

Comment:
``args.Length`` returns the number of JavaScript arguments that were passed over. It should prevent, that no argument gets passed in. Other io.js APIs handle this special case with a `TypeError`, too. `RangeError` typically thrown if e.g. the `process.uid` is out of the possible range. My main thought was to keep it in sync with the other APIs.